vcpkg.json

```diff
@@ -1,20 +1,31 @@
 {
-    "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
-    "name": "dosbox-staging",
-    "description": "DOSBox Staging is a modern continuation of DOSBox with advanced features and current development practices.",
-    "version": "0.82.0-alpha",
-    "dependencies": [
-        "fluidsynth",
-        "gtest",
-        "iir1",
-        "libmt32emu",
-        "libpng",
-        "opusfile",
-        "sdl2",
-        "sdl2-image",
-        "sdl2-net",
-        "speexdsp",
-        "tracy",
-        "zlib"
-    ]
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
+  "name": "dosbox-staging",
+  "description": "DOSBox Staging is a modern continuation of DOSBox with advanced features and current development practices.",
+  "version": "0.81.0-alpha",
```

Comment:
Why are we changing the version from 0.82 alpha to 0.81?

Reply:
oversight on my part, I copied that snippet in from @shermp's `libslirp` fix, good catch.